ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -1371,6 +1371,38 @@ presubmits:
           requests:
             cpu: 10m
     trigger: (?m)^/test( | .* )pylint,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build03
+    context: ci/prow/sync-rover-groups
+    decorate: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-release-sync-rover-groups
+    optional: true
```

Comment:
Let's make it:

```
always_run: false
run_if_changed: core-services/sync-rover-groups/_config.yaml
```

Context is in https://github.com/openshift/release/pull/25651:

> alvaroaleman pointed out that o/release presubmits likely generate a lot of GH API traffic in crier. We could optimize this with a reasonable usage of run_if_changed: the repository is partitioned quite nicely and most presubmits deal with only one partition. This also lessens the load in CI overall, makes o/release PRs less cluttered...

Reply:
We finally reach the point where we do not afford running everything on every PR.

But this presubmit checks the violations such as define groups in the `clusters` folder.
Config file should not matter when `--validate-subjects`.

Let me remove it.